fix(docs-infra): make external icons in the middle

ajitsinghkaler opened the pull request:
external icons sometimes are place in wrong position in title
adding them to the middle so that they don't seem like a
subscript.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

![image](https://user-images.githubusercontent.com/39260684/89912403-24033980-dc10-11ea-80b5-6d423c406d69.png)


Issue Number: N/A


## What is the new behavior?
![image](https://user-images.githubusercontent.com/39260684/89912557-50b75100-dc10-11ea-820a-e4e5e13975ee.png)



## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview 5d89ef4 at https://pr38410-5d89ef4.ngbuilds.io/.

mary-poppins commented:
You can preview 7f73038 at https://pr38410-7f73038.ngbuilds.io/.

gkalpak reviewed (changes requested):
Looks reasonable. (Not ideal, but better than what we have atm.)
Can you please update the commit message to:
1. Be more clear (for example, "fix vertical alignment of external link icons").
2. Mention where the broken behavior can be observed (for example, link to https://github.com/angular/angular/blame/8366effeec8e74c1a11d2a6e1a48d42e849e4653/aio/content/guide/roadmap.md#L37).

ajitsinghkaler committed:
fix(docs-infra): fix vertical alignment of external link icons

At some places external link icons appear as a subscript. For example
https://github.com/angular/angular/blame/8366effeec8e74c1a11d2a6e1a48d42e849e4653/aio/content/guide/roadmap.md\#L37
this commit places external link icons in the middle to improve there
positioning in a line.

mary-poppins commented:
You can preview f028545 at https://pr38410-f028545.ngbuilds.io/.

mary-poppins commented:
You can preview c47e707 at https://pr38410-c47e707.ngbuilds.io/.
